CMakeLists.txt

```diff
@@ -342,5 +342,31 @@ if(UNIX AND NOT APPLE)
   endif(FREECIV_ENABLE_RULEDIT)
 endif(UNIX AND NOT APPLE)
 
+# We grab the Libertine Font package online for non-Unix (e.g. Win32, MSYS2, and macOS)
+#  For Linux, we add a dependency to the package.
+if(NOT UNIX)
+  message(STATUS "Downloading Linux Libertine Font Package")
+
+  include(ExternalProject)
+  ExternalProject_Add(Libertine
+    PREFIX ${CMAKE_BINARY_DIR}
+    LOG_DIR ${CMAKE_BINARY_DIR}
+    DOWNLOAD_NO_PROGRESS TRUE
+    DOWNLOAD_EXTRACT_TIMESTAMP TRUE
+    TIMEOUT 30
+    CONFIGURE_COMMAND ""
+    BUILD_COMMAND ""
+    INSTALL_COMMAND ""
+    URL https://sourceforge.net/projects/linuxlibertine/files/linuxlibertine/5.3.0/LinLibertineOTF_5.3.0_2012_07_02.tgz
+  )
+
+  install(
+    DIRECTORY ${CMAKE_BINARY_DIR}/src/Libertine
+    DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/fonts
```

Comment:
This installs to `/usr/share/fonts` on Linux and could overwrite some system files. Better put them in our private data dir, `SHARE/freeciv21/fonts`.